fvm/environment/account_info.go

```diff
@@ -241,7 +243,8 @@ func (info *accountInfo) GetAccount(
 	}
 
 	if info.serviceAccountEnabled {
-		balance, err := info.GetAccountBalance(common.Address(address))
+		balance, err := info.GetAccountBalance(
+			common.MustBytesToAddress(address.Bytes()))
```

Comment:
A `flow.Address` is `[8]byte`, and so is `common.Address` – why change this?

The new code will allocate and convert to a slice, then just convert it back to an array

Reply:
i'm planning to change flow.Address's type representation from [8]byte to a string to enable it to compose with flow.RegisterId (the flow register's address could be an empty string or an 8 byte string).  This is prep work for that